crates/doc/src/ptr.rs

```diff
@@ -9,6 +9,8 @@ pub enum Token {
     Index(usize),
     /// JSON object property name without escaping. Never an integer.
     Property(String),
+    // Schema representing an object's additionalProperties
```

Comment:
I think this should be `NextProperty`, paralleling `NextIndex`. That's because it's a conceptual placeholder for the next property to be added.

It might happen to sometimes be schematized through a `additionalProperties` JSON schema keyword, but does not have any literal association. This is a JSON Pointer, which has zero dependence on the JSON Schema spec.